easybuild/tools/github.py

```diff
@@ -395,6 +407,235 @@ def post_comment_in_issue(issue, txt, repo=GITHUB_EASYCONFIGS_REPO, github_user=
         raise EasyBuildError("Failed to create comment in PR %s#%d; status %s, data: %s", repo, issue, status, data)
 
 
+def _copy_easyconfigs_to_repo(paths, target_dir):
```

Comment:
It's merely a convention that indicates that this is a 'private' function, i.e. that it shouldn't be used outside of this module. It's nothing more than a naming convention though, it'll work either way.